internal/cli/apps.go

```diff
@@ -78,19 +77,16 @@ var CmdAppList = &cobra.Command{
 		}
 
 		err = client.Apps(all)
-		if err != nil {
-			return errors.Wrap(err, "error listing apps")
-		}
-
-		return nil
+		return errors.Wrap(err, "error listing apps")
```

Comment:
I had to remember that `errors.Wrap` returns `nil` if the `err` is `nil itself. Maybe add a small line about that before the command. Something like
> // __Note__: errors.Wrap (nil, "...") == nil

Not just here, all the changed places.